extensions/amp-accordion/1.0/accordion.js

```diff
@@ -48,6 +48,13 @@ const CHILD_STYLE = {
   position: 'relative',
 };
 
+const SECTION_HEADER_STYLE = {
+  cursor: 'pointer',
+  backgroundColor: '#efefef',
```

Comment:
Cursor pointer is ok here. But we can't put any of the rest into the inline style - this would make them un-overridable in stylesheets. Let's remove these styles for now. If the wg decides we want these styles here, we will have to bring JSS back. But for now it's easier to not have it, imho.